Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -138,6 +140,164 @@ public class PushDeploymentController : ApiController
             }
         }
 
+        [HttpPost]
+        [HttpPut]
+        public async Task<HttpResponseMessage> OnePushDeploy(
```

Comment:
>OnePushDeploy [](start = 47, length = 13)

This method is too big. Makes it difficult to read for someone looking at this for the first time. Is there a way to fix that?